main.go

```diff
@@ -176,6 +190,43 @@ func main() {
 		setupLog.Error(err, "unable to setup controller", "controller", "Subscription")
 		os.Exit(1)
 	}
+
+	// Client to use before cache is created
+	restConfig := ctrl.GetConfigOrDie()
+	restConfig.Timeout = time.Second * 10
+
+	client, err := k8sclient.New(restConfig, k8sclient.Options{
+		Scheme: mgr.GetScheme(),
+	})
+
+	if err != nil {
+		setupLog.Error(err, "unable to create client for checking if Addon Operator is installed", "controller", "AddonInstance")
+		os.Exit(1)
+	}
+
+	// Check is addon operator installed
+	addonOperatorInstalled, err := status.IsAddonOperatorInstalled(client)
+	if err != nil {
+		setupLog.Error(err, "unable to check is addon operator installed", "controller", "AddonInstance")
+		os.Exit(1)
+	}
+
+	// Only start controller if addon operator is installed and is not a sandbox install
+	if addonOperatorInstalled && !isSandbox {
```

Comment:
I assumed we do not want to start the controller on sandbox installs

Reply:
@KevFan I think this is probably safe. RHOAM on the current stg and prod sandbox clusters is already installed by CatalogSource anyways.